docassemble/AppealsMotionToReconsider/data/questions/Motion_to_Reconsider.yml

```diff
@@ -133,23 +133,45 @@ comment: |
   This question is used to introduce your interview. Please customize
 id: Motion to Reconsider
 question: |
+  Motion to Reconsider, Motion to Modify, or Motion to Reconsider and Modify
+subquestion: |
+  File one of these motions if you need the Appeals Court to look at your case again because:  
+  
+  * the court's reasoning that led to the current order was wrong, 
+  * the court failed to consider evidence, or
+  * the judge should know about new information that affects the decision the court already made. 
+  
   What do you need help with?
 fields: 
   - no label: needs_help_with
     datatype: checkboxes
     choices: 
-      - Reconsideration: Reconsideration
-      - Modification: Modification
-      - A real-world change of circumstances: A real-world change of circumstances
-      - Changing the actual outcome of the decision: Changing the actual outcome of the decision
-      - Changing arguments made in court: Changing arguments made in court
+      - Reconsideration.: Reconsideration
+        help: If you need the Court to undo or change the previous decision in your Appeal.
+      - Modification.: Modification
+        help: If you need the court to correct some inaccuracy in the decision keep the outcome the same.
+      - A real-world change of circumstances.: A real-world change of circumstances
+        help: needs more explanation
+      - Changing the actual outcome of the decision.: Changing the actual outcome of the decision
+        help: needs more explanation
+      - Changing arguments made in court.: Changing arguments made in court
     none of the above: False
 ---
+code: |
+  if needs_help_with["Reconsideration"]:
+    motion_type = "Motion for Reconsideration"
+  elif needs_help_with["Modification"]:
+    motion_type = "Motion for Modification"
+  elif needs_help_with.all_true('Reconsideration','Modification'):
+    motion_type = "Motion for Reconsideration and Modification"
+  else:
+    motion_type = "Can't file a motion"
```

Comment:
The user can still file their motion, but we just warn them that it will likely not be successful.